pyaedt/application/Variables.py

```diff
@@ -452,7 +496,7 @@ class VariableManager(object):
      'Var3': <pyaedt.application.Variables.Expression at 0x2661f34cb48>,
      '$PrjVar1': <pyaedt.application.Variables.Expression at 0x2661f34cc48>}
 
-    >>> # get a dictionary of just the design variables
+    >>> # Get a dictionary of just the design variables
```

Comment:
```suggestion
    Get a dictionary of just the design variables

```

This can be rendered as plain text since it's just description text.